### DISCUSSION TITLE
How do i use the exported JSON API link ?

### DISCUSSION THREAD

**Discussion by AliAlAbrash** [Budibase/budibase#2755] [Help] [answered]
Hello, i am currently trying to figure out how to make use of the API link, that i receive by exporting the JSON of a table.
could you help me out pls? Thank you in advance

**Comment by mjashanks** [comment, marked as answer]
Hi @AliAlAbrash - this sounds like a bug - could you give an example? When I export my table, I get a full JSON representation of my data.

Are you using Budibase internal tables or a SQL table?

**Reply by AliAlAbrash** [reply]
Hi there! Thanks for the quick answer. Its the same result with either an SQL table or a budibase table. But lets say i want to use SQL. My Goal is to receive a link that lets me read the live JSON when inserted into the browser url.  Here are some screenshots:

![export](https://user-images.githubusercontent.com/89005959/134879120-6baf7fa2-d16d-40ac-8336-da538690e174.png)

![Screenshot 2021-09-27 104331](https://user-images.githubusercontent.com/89005959/134879158-bd9024b2-9927-4a2c-99e6-77f188c76156.png)

![api](https://user-images.githubusercontent.com/89005959/134879189-0b837f58-ecda-4132-9581-3efa7ae8df3d.png)

**Reply by mjashanks** [reply]
Ah - I can reproduce this in the currently released version of Budibase, but not in the development version. So, I think that this is a bug that will be fixed and released in a few days.

**Reply by AliAlAbrash** [reply]
That sounds great. Thanks!
